apps/demo/config/vite/app.ts

```diff
@@ -41,6 +41,16 @@ export default defineConfig({
       },
       runtimeOptions: {
         data: {
+          themes: [
+            {
+              id: 'light-default',
```

Comment:
maybe we can get this from the JS tokens? It is not there yet, but the exporter will return it in the future.